lib/Frontend/OutputFileMap.cpp

```diff
@@ -108,21 +102,47 @@ void OutputFileMap::dump(llvm::raw_ostream &os, bool Sort) const {
   }
 }
 
-bool OutputFileMap::parse(std::unique_ptr<llvm::MemoryBuffer> Buffer,
-                          StringRef workingDirectory) {
+void OutputFileMap::write(llvm::raw_ostream &os,
+                          ArrayRef<StringRef> inputs) const {
+  for (const auto input : inputs) {
+    const TypeToPathMap *outputMap = getOutputMapForInput(input);
+    if (!outputMap)
+      continue;
+    os.write_escaped(input);
```

Comment:
You still need quotes around the calls to `write_escaped`, sorry.

Reply:
Thanks.

Reply:
You *still* need quotes around the call to `write_escaped`. ;-) (You got the one below but this one was missed.)